.github/workflows/build.yml.in

```
@@ -61,13 +61,14 @@ jobs:
       - name: "check the formatting of the PR title and for missing or inconsistent labels"
         # TODO: skip this check if the PR is in draft stage!
         shell: bash # just in case
+        env:
+          TITLE: ${{ github.event.pull_request.title }}
         run: |
           set -eo pipefail
-          title="${{github.event.pull_request.title}}"
-          echo "$title"
+          echo "$TITLE"
           label_names=$(echo "${{ toJson(github.event.pull_request.labels) }}" | jq -r '.[].name')
           echo "$label_names"
-          lake exe check-title-labels $title $label_names
+          lake exe check-title-labels $TITLE $label_names
```

Ah, now that I actually try to parse the position indicators in the error messages, it looks like there are still quoting issues in this line:
```suggestion
          lake exe check-title-labels "$TITLE" "$label_names"
```